src/Corax/IndexWriter.cs

```diff
@@ -1851,6 +1890,38 @@ void ThrowTriedToDeleteTermThatDoesNotExists()
             entriesForTerm.Dispose(_entriesAllocator);
 
             _indexMetadata.Increment(indexedField.NameTotalLengthOfTerms, totalLengthOfTerm);
+
+            long TryGetNullTermPostingList(CompactTree tree, out long termContainerId, out CompactTree.CompactKeyLookup keyLookup, out bool termExists, ref EntriesModifications entries, out bool alreadyIndexed)
+            {
+                keyLookup = default;
+                termContainerId = ~long.MaxValue | (tree.RootPage << 3); // sign bit means its null, rootpage identificates field
+                termExists = true;
+                alreadyIndexed = false;
+                //We want to obtain posting lists for NULL value from different tree than CompactTree to make rest code as valid ;-)
+                //We will
+                var postingList = _nullEntriesPostingLists.ReadInt64(indexedField.Name);
```

Comment:
For null, we should _always_ just use a posting list.
Worst case scenario here is that we have a `null` posting list per field?
Not worth the effort to allocate for it.
Also means that we can _know_ what the value is statically.